h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -1397,6 +1397,37 @@ def equal_two_arrays(array1, array2, eps, tolerance):
         print("The two arrays are of different size!")
         sys.exit(1)
```

Comment:
you should use assertions instead of `sys.exit(1)`. If you use `exit` then all test infrastructure will be affected - no proper output from tests, partial test result, ...